### PR TITLE
Awesome books plain java script with objects

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this PR we did the following:

- Create html file
- Create a collection that keeps a list of books 
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Add Display function to display books saved in the collection in the top part of the page.
- Add code to save data  in the browser's memory

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @shahira-sadat, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that has linter errors, kindly fix it and then ask for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  const newBook = { title, author, id };
+
+  if (getExistingBooks()) {
+    getExistingBooks().forEach((existingBook) => {
+      books.push(existingBook);
+    });
+  }
+
+  books.push(newBook);
```
Comment (vechicin) [inline main.js:31]: [OPTIONAL] Consider using methods that don't mutate the array.

Screen Shot 2022-06-27 at 16 48 36

File: main.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    });
+  }
+};
+
+displayBooks();
+
+function removeBook(bookId) {
+  const filterBooks = getExistingBooks().filter((existingBook) => existingBook.id !== bookId);
+  saveToLocalStorage(filterBooks);
+  window.location.reload();
```
Comment (vechicin) [inline main.js:65]: [OPTIONAL] Consider not reloading the page after book submission, it'll improve site performance.

**Review by vechicin — APPROVED** [review]
# Hi @shahira-sadat and @sja-thedude ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Nice `id` generator!
- [x] Organized codebase
- [x] Linters passing locally

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**